src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -8,7 +8,13 @@ import { OHRIFormLauncherWithIntent } from '../ohri-form-launcher/ohri-form-laun
 import styles from './encounter-list.scss';
 import OTable from '../data-table/o-table.component';
 import { Button, Link, OverflowMenu, OverflowMenuItem, Pagination } from 'carbon-components-react';
-import { encounterRepresentation, stopReasonUUID, substituteReasonUUID, switchReasonUUID } from '../../constants';
+import {
+  encounterRepresentation,
+  stopReasonUUID,
+  substituteReasonUUID,
+  switchReasonUUID,
+  restartReasonUUID,
+} from '../../constants';
```

Comment:
@kajambiya like we discussed earlier today, you don't need to touch this file, this file is getting bloated